zenoh-plugin-ros2dds/src/node_info.rs

```diff
@@ -56,14 +66,24 @@ pub struct MsgSub {
     pub name: String,
     #[serde(rename = "type")]
     pub typ: String,
+    // List of DDS Readers declared by 1 Node for a same topic
+    // Issue #27: usually only 1 Reader, but may happen that 1 Node declares several Subscribers
+    //            on the same topic. In this case `ros2 node info <node_id>` still shows only 1 Subscriber,
+    //            but all the writers can be seen with `ros2 topic info <topic_name> -v`.
+    //            Hence the choice here to aggregate all the Readers in 1 Subscriber representation.
+    //            The Subscriber is declared undiscovered only when all its Readers are undiscovered.
```

Comment:
General question, 
What are the differences between Readers and Subscribers In this text ? 
Are Subscribers a Zenoh concept, and Readers a ROS2 concept ?

Reply:
Answering on behalf of @JEnoch . Here *Readers* are related to DDS while *Subscribers* are related to Zenoh.

Reply:
To be more precise, in the context of this code:
- *Reader* means *DDS DataReader*
- *Subscriber* means *ROS 2 Subscriber*, which happens to map in this plugin to a Zenoh Subscriber.